src/libs/cropOrRotateImage/index.native.ts

```diff
@@ -0,0 +1,18 @@
+import RNImageManipulator from '@oguzhnatly/react-native-image-manipulator';
+import RNFetchBlob from 'react-native-blob-util';
+import {CropOrRotateImage} from './types';
+
+/**
+ * Crops and rotates the image on ios/android
+ */
+const cropOrRotateImage: CropOrRotateImage = (uri, actions, options) =>
```

Comment:
This seems a little weird to me. Any reason not to declare the types inline?


```ts
function cropOrRotateImage(uri: string, actions: Action[], options: CropOrRotateImageOptions): Promise<FileWithUri | RNManipulator > {
  ...
}
```

Reply:
@thiagobrez when we have platform specific files we need to follow this [pattern](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization). Thats why its like that

Reply:
@kubabutkiewicz Cool! 👍🏻